templates/contest/list.html

```diff
@@ -124,7 +124,7 @@
 {% macro contest_join(contest, request) %}
     {% if not request.in_contest %}
         <td>
-            {% if request.profile in contest.organizers.all() %}
+            {% if request.profile in contest.authors.all() or request.profile in contest.curators.all() %}
```

Comment:
```suggestion
            {% if request.profile in contest.authors.all() or request.profile in contest.curators.all() or request.profile in contest.testers.all() %}
```
I think we need testers here as well to be consistent with the contest detail page. (not sure if there's a better way to write this line than a triple `or`)